<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1242</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1221</width>
      <height>391</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>991</width>
      <height>391</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Добавить</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>361</width>
        <height>311</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить заказ</string>
      </property>
      <widget class="QPushButton" name="pb_addProject">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>280</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>139</width>
         <height>241</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>ID сотрудника</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_10">
          <property name="text">
           <string>ID клиента</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>Дата открытия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>Дата закрытия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Затраченное время</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_14">
          <property name="text">
           <string>ПО для создания</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_15">
          <property name="text">
           <string>ПО для рендера</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Вид заказа</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>30</y>
         <width>181</width>
         <height>260</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <widget class="QLineEdit" name="le_projectCreatorID"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_projectClientID"/>
        </item>
        <item>
         <widget class="QDateEdit" name="de_projectOpenDate">
          <property name="date">
           <date>
            <year>2013</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="de_projectCloseDate">
          <property name="date">
           <date>
            <year>2013</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
          <property name="displayFormat">
           <string>yy-MM-dd</string>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sb_projectTime"/>
        </item>
        <item>
         <widget class="QComboBox" name="cb_projectCreationProg">
          <item>
           <property name="text">
            <string>Blender</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Maya</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>3ds Max</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Houdini</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_projectRenderProg">
          <item>
           <property name="text">
            <string>RenderMan</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MentalRay</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>V-Ray</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>DreamScape</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cb_projectSendForm">
          <item>
           <property name="text">
            <string>Model</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Animation</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pictures</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>20</y>
        <width>251</width>
        <height>181</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить сотрудника</string>
      </property>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>30</y>
         <width>81</width>
         <height>91</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Должность</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>131</width>
         <height>95</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLineEdit" name="le_creatorSurname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_creatorName"/>
        </item>
        <item>
         <widget class="QComboBox" name="cb_creatorPosition">
          <item>
           <property name="text">
            <string>3d artist</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Modeller</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Animator</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
       <zorder>cb_creatorPosition</zorder>
       <zorder>le_creatorSurname</zorder>
       <zorder>le_creatorName</zorder>
      </widget>
      <widget class="QPushButton" name="pb_addCreator">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>140</y>
         <width>98</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>371</width>
        <height>251</height>
       </rect>
      </property>
      <property name="title">
       <string>Добавить клиента</string>
      </property>
      <widget class="QPushButton" name="pb_addClient">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>220</y>
         <width>97</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Добавить</string>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>40</y>
         <width>261</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLineEdit" name="le_clientSurname">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_clientName">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_clientAddress">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_clientEmail">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_clientPhone">
          <property name="minimumSize">
           <size>
            <width>251</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>251</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>68</width>
         <height>161</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Фамилия</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Имя</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Адрес</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>E-mail</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Телефон</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Удалить</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>181</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Удалить клиента</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>184</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_8">
        <item>
         <widget class="QRadioButton" name="rb_delClientByID">
          <property name="text">
           <string>По ID</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delClientID"/>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_delClientByName">
          <property name="text">
           <string>По ФИО</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delClientSurname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delClientName"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_delClient">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>30</y>
        <width>181</width>
        <height>231</height>
       </rect>
      </property>
      <property name="title">
       <string>Удалить сотрудника</string>
      </property>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>184</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QRadioButton" name="rb_delCreatorByID">
          <property name="text">
           <string>По ID</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delCreatorID"/>
        </item>
        <item>
         <widget class="QRadioButton" name="rb_delCreatorByName">
          <property name="text">
           <string>По ФИО</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delCreatorSurname"/>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delCreatorName"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_delCreator">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>30</y>
        <width>181</width>
        <height>161</height>
       </rect>
      </property>
      <property name="title">
       <string>Удалить проект</string>
      </property>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>40</y>
         <width>151</width>
         <height>111</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <widget class="QRadioButton" name="rb_delProjectByID">
          <property name="text">
           <string>По ID</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_delProjectID"/>
        </item>
        <item>
         <widget class="QPushButton" name="pb_delProject">
          <property name="text">
           <string>Удалить</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>240</y>
        <width>113</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>280</y>
        <width>97</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Поиск и отчёты</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QPushButton" name="pb_showTable">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>620</y>
      <width>131</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Show client table</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>500</y>
      <width>141</width>
      <height>108</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QRadioButton" name="rb_client">
       <property name="text">
        <string>Client</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_creator">
       <property name="text">
        <string>Creator</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="rb_project">
       <property name="text">
        <string>Project</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chb_replaceIDs">
       <property name="text">
        <string>Replace IDs</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1242</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
